### ISSUE TITLE
Date Range - allowEmptyValue story has accessibility violation - Input has no label

### ISSUE THREAD

**Issue by szeyingyau** [Sage/carbon#4570]
### Current behaviour

The `allowEmptyValue` story of the Date Range component has an accessibility violation - "Ensures every form element has a label" as the date inputs do not have any labels.

Please see https://carbon.sage.com/?path=/story/date-range--allow-empty-value

### Expected behaviour

There are no accessibility violations in the `allowEmptyValue` story

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/date-range--allow-empty-value

### Carbon Version

99.0.0

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
FE-4478
